module/zfs/arc.c

```diff
@@ -3898,7 +3898,7 @@ arc_evict_hdr(arc_buf_hdr_t *hdr, kmutex_t *hash_lock, uint64_t *real_evicted)
 {
 	arc_state_t *evicted_state, *state;
 	int64_t bytes_evicted = 0;
-	int min_lifetime = HDR_PRESCIENT_PREFETCH(hdr) ?
+	uint32_t min_lifetime = HDR_PRESCIENT_PREFETCH(hdr) ?
```

Comment:
Why uint32_t here?  Both values on right are uint_t and the consumers are uint_t or hrtime_t, none are uint32_t.

Reply:
Fixed.